sql/updates/reverse-dev.sql

```diff
@@ -75,3 +75,29 @@ BEGIN
     );
 END
 $$ LANGUAGE 'plpgsql';
+
+--undo compression feature for caggs
+--check that all continuous aggregates have compression disabled
+--this check is sufficient as we cannot have compressed chunks, if
+-- compression is disabled
+DO $$
+DECLARE
+  cagg_name NAME;
+  matht_name NAME;
+  matht_schema NAME;
+BEGIN
+    FOR cagg_name , matht_schema, matht_name IN
+        SELECT view_name, 
+               materialization_hypertable_schema ,
+               materialization_hypertable_name   
+        FROM timescaledb_information.continuous_aggregates
+        WHERE compression_enabled is TRUE
```

Comment:
IMHO we can save all caggs with compression in an array and then show to the user the complete list, something that we did here already: https://github.com/timescale/timescaledb/commit/958040699c4986a98fab788bafd599d24281f1d8#diff-328ad2a463009f63aded2e997441ab13d9f7e725b14cc68e68fb807a78984e7bR10